src/extension.ts

```diff
@@ -142,6 +149,22 @@ export async function activate(context: vscode.ExtensionContext): Promise<vscode
 	return Promise.resolve(context);
 }
 
+export async function _isApexGuruEnabledInOrg(): Promise<boolean> {
+	try {
+		const connection = await CoreExtensionService.getConnection();
+		const response:ApexGuruAuthResponse = await connection.request({
+			method: 'GET',
+			url: Constants.APEX_GURU_AUTH_ENDPOINT,
```

Comment:
Oh this is interesting... the CoreExtensionService's connection only requires relative url paths? That's nice.

Reply:
Yes :)

Reply:
Just to add more details - this is what we get by leveraging sfdx core instead of jsforce or axios. It automatically picks up the base url from the sfdx-project.json file.